src/azure-cli/azure/cli/command_modules/synapse/_help.py

```diff
@@ -1438,7 +1438,7 @@
   - name: Create a notebook.
     text: |-
         az synapse notebook create --workspace-name testsynapseworkspace \\
-          --name testnotebook --file @path/notebook.json
+          --name testnotebook --file @path/notebook.ipynb
```

Comment:
Does that mean `.json` is preferred for notbook file? Can `.ipynb` file still work well?
I remember @idear1203 insisting on `.ipynb`. (See https://github.com/Azure/azure-cli/pull/17661#issuecomment-818489392)

Reply:
@evelyn-ys , thanks for confirmation. Yes, we still want to use `.ipynb` in this command parameter. The reason of change here is, we want to correct the documentation to make it clear to our customers that `.ipynb` is the desired file format.

Reply:
Pinging @evelyn-ys , thanks a lot!